src/test/java/org/jabref/logic/cleanup/RemoveLinksToNotExistentFilesTest.java

```diff
@@ -64,14 +83,53 @@ void setUp(@TempDir Path bibFolder) throws IOException {
 
     @Test
     void deleteLinkedFile() {
+
+        assertEquals(2, entry.getFiles().size()); //Entry has 2 linked files
+
         fileBefore.toFile().delete();
+        
         List<FieldChange> changes = removeLinks.cleanup(entry);
+        
         assertFalse(changes.isEmpty());
```

Comment:
I suspect that we may get feedback on these kinds of asserts. Rather than just checking that there are changes being returned. The ideal would be to expect some change: 
```
List<FieldChange> expectedChanges = ...
assertEquals(expectedChanges, changes); 
```
They should be comparable through the equals method in FieldChange.java otherwise maybe we could use the toString() method.

Reply:
I think the assertion with expectedEntry already does this